jest.config.ts

```diff
@@ -9,4 +9,4 @@ const jestConfig: JestConfigWithTsJest = {
   projects: ['<rootDir>/packages/plainjs', '<rootDir>/packages/node', '<rootDir>/packages/whatwg'],
 };
 
-export default jestConfig;
\ No newline at end of file
+export = jestConfig;
```

Comment:
What's the difference between `export default` & `export equals`?

Reply:
See documentation of the `verbatimModuleSyntax ` option (link was provided in the description):

<img width="726" alt="Screenshot 2023-08-24 at 05 45 54" src="https://github.com/juanjoDiaz/streamparser-json/assets/72159681/28080db0-8737-4dd2-b25e-d68d6a2414ad">